coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -20,24 +20,26 @@
 @register_op(doc_str="")
 class cond(Operation):
     """
-    Conditional execution. The return types must be identical between the true
-    and false branches.
+    Perform a conditional execution. The return types must be identical
+    between the true and false branches.
 
     Parameters
     ----------
     pred: tensor<[], bool> (Required)
-        * 0D tensor (scalar) predicate to switch between true and fall branches.
+        * 0D tensor (scalar) predicate to switch between true and false branches.
 
-    _true_fn: Python function (Required)
-	* A Python function that will be executed if ``cond`` evaluates to ``True``. It must take 0 input and return one or more values, whose types will be taken to be the return type of the operation.
+    _true_fn: function (Required)
+        * A Python function that executes if ``cond`` evaluates to ``True``.
+        * It must take ``0`` as input, and return one or more values whose type becomes the operation's return type.
```

Comment:
`0` as input or no inputs? Can you please clarify this part?

Reply:
The previous phrase I'm working with is "It must take 0 input". If that meant "no inputs" the phrase was misleading in the first place. I assume it means "It must take ``0`` as input".